db/migrate/20190216224107_add_food_id_to_foods.rb

```diff
@@ -1,5 +0,0 @@
-class AddFoodIdToFoods < ActiveRecord::Migration[5.2]
```

Comment:
Why are you deleting this migration? Didn't you already run it on the last branch?

Reply:
I was looking at my schema and I didn't actually see food_id there. I seemed to remember that I ended up not needing it. Is it normally not a good practice to delete migrations? I can put it back.

Reply:
I seem to have 2 of the same migrations. Same name, just created to different times. Just the hash that gets inserted to have a unique name is different. So, this is there 20190216224107_add_food_id_to_foods.rb

Reply:
Hmm... I see. Yeah, it's normally best practice to leave all migrations in place after they've been run, but if you have a dupe I guess that makes sense.